pkg/proto/finalization.go

```diff
@@ -0,0 +1,106 @@
+package proto
+
+import (
+	"encoding/binary"
+
+	g "github.com/wavesplatform/gowaves/pkg/grpc/generated/waves"
+)
+
+const heightSize = 4
```

Comment:
I think this constant should be local inside `EndorsementMessage` method. No need to do it package private. Also you can use `uint32Size` constant from `proto` package.